app/containers/ExplorePage.tsx

```diff
@@ -51,45 +52,110 @@ function chooseFile(
 export default function ExplorePage() {
   const { t } = useTranslation();
   const [filePath, setFilePath] = useState<string>();
+  const [filters, setFilters] = useState<object>({
+    activeAnimals: [],
+    activeCameras: [],
+    activeStations: [],
+    activeChecks: []
+  });
   const [data, setData] = useState<undefined | ObservationsData>();
+  const handleFilters = val => {
+    setFilters({ ...filters, ...val });
+  };
+
+  const filterCondition = (needle, haystack) => {
+    if (haystack.length === 0) return true;
+    return haystack.map(entry => entry.value).includes(needle);
+  };
+
+  const getFilteredData = options => {
+    let filtered = options.observations;
+
+    if (typeof filters !== 'undefined') {
+      filtered = filtered.filter(entry => {
+        const validEntry =
+          filterCondition(entry.pred_1, filters.activeAnimals) &&
+          filterCondition(entry.camera, filters.activeCameras) &&
+          filterCondition(entry.station, filters.activeStations) &&
+          filterCondition(entry.check, filters.activeChecks);
+        return validEntry;
+      });
+    }
+    return { observations: filtered };
+  };
+
+  const MainPanel: React.SFC = () => (
+    <div style={{ display: 'flex' }}>
+      <div
+        style={{
+          flex: '1',
+          width: '55%',
+          paddingBottom: '20px',
+          marginRight: '10px'
+        }}
+      >
+        <Card style={{ height: '100%' }} interactive elevation={Elevation.TWO}>
+          <AnimalsPlot data={getFilteredData(data)} />
+        </Card>
+      </div>
+      <div
+        style={{
+          flex: '1',
+          width: '45%',
+          paddingBottom: '20px',
+          marginLeft: '10px'
+        }}
+      >
+        <Card style={{ height: '100%' }} interactive elevation={Elevation.TWO}>
+          <Map data={getFilteredData(data)} />
+        </Card>
+      </div>
+    </div>
+  );
+  const TablePanel: React.SFC = () => (
+    <ObservationsTable style={{ width: '100%' }} data={getFilteredData(data)} />
+  );
 
   const contents =
     data !== undefined ? (
       <>
-        <h1>{t('Explore')}</h1>
-        <Button
-          text={t('Back')}
-          icon="arrow-left"
-          onClick={() => setData(undefined)}
-          style={{ marginBottom: '10px', backgroundColor: '#fff' }}
-        />
-        ,
-        {/* <div style={{ flex: 1, paddingBottom: '20px' }}>
-          <Card interactive elevation={Elevation.TWO}>
-            <h2 style={{ marginTop: 0 }}>{t('Filters')}</h2>
-          </Card>
-        </div> */}
-        ,
-        <div style={{ flex: 1, width: '100%', paddingBottom: '20px' }}>
-          <Card interactive elevation={Elevation.TWO}>
-            <AnimalsPlot data={data} />
-          </Card>
+        <div>
+          <h1>{t('Explore')}</h1>
+        </div>
+        <ExplorerFilter data={data} onChange={handleFilters} />
+        <div style={{ minHeight: '30px' }}>
+          <span>
```

Comment:
it would be nice to render these statistics in some boxes similar to [_statistic_ ](https://semantic-ui.com/views/statistic.html)from semanticui:

![image](https://user-images.githubusercontent.com/1421503/83498421-30567200-a4bc-11ea-9b7c-56ea9c1c798f.png)

i think that the closes in Blueprint.js is the [Card component](https://blueprintjs.com/docs/#core/components/card) and you could use it to build that. Example of something very similar is on the home page of the Blueprint documentation:

![image](https://user-images.githubusercontent.com/1421503/83498617-6c89d280-a4bc-11ea-80de-d14b43144faf.png)

Reply:
Switched to cards and added more metrics